index.md

```diff
@@ -28,7 +28,7 @@ Paragraph(s) of introductory material.
 8.  Switch to a new stripping (20 -> 21) for one line in MC
 9.  Split your jobs into subjobs
 10. (Stripping line efficiency)
-11. How to use a stripping line for a slightly different decay
+11. [How to use a stripping line for a slightly different decay](14-rerun-stripping.html)
```

Comment:
Originally we thought that this line would be taking a candidate from an existing stripping line and replacing a mass hypothesis (eg. D->pipi gets changed to D->Kpi). So I would link this under line 28